src/header/header.scss

```diff
@@ -154,7 +154,8 @@
     @include iui-header-button-icon;
   }
 
-  .iui-header-button.iui-header-button { // boost specificity
+  .iui-header-button.iui-header-button {
+    // boost specificity
```

Comment:
Remove or move to previous line

Reply:
Wasn't prettier that made these changes? Maybe need to fix rules there?

Reply:
well, prettier is not configurable so we dont have a choice

Reply:
What you mean not configurable? Also, @gretanausedaite have you ran `yarn install` before working on buttons? I am wondering was it prettier that made the changes or Greta.

Reply:
prettier is not configurable! There is no way to tell prettier to keep that comment on the same line as it does not like anything after `{`.

![image](https://user-images.githubusercontent.com/9084735/142436209-93f598e6-e184-4a87-9beb-f6a9b94f0425.png)

Reply:
Ok, then I don't understand why prettier changed comment placement here and not on the PR when you added prettier to the pre-commit. And if it was the prettier that changed it then no one can blame Greta for this comment movement 🤷‍♂️

Reply:
I think it's because this file was not staged in my PR? Not sure.

And no one is blaming Greta 😆 but we do need to fix the formatting, as this comment does not make sense on the next line

Reply:
I thought you ran prettier through all the files... Then there can be more random changes like this.

Reply:
Can't we just add comments before statements and not after them if prettier just keeps "fixing" them? 
This doesn't look bad:

// boost specificity
.iui-header-button.iui-header-button {

Reply:
that's something I also suggested for this
![image](https://user-images.githubusercontent.com/9084735/142444185-e11f623a-2e37-4b9a-b933-c143e227a14e.png)

Reply:
Got `husky > pre-commit` error while trying to push code with fix. 
`--no-verify` command worked, so should be same line now.